htdocs/takepos/invoice.php

```diff
@@ -503,6 +503,9 @@ function fail($message)
 	$localtax1_tx = get_localtax($tva_tx, 1, $customer, $mysoc, $tva_npr);
 	$localtax2_tx = get_localtax($tva_tx, 2, $customer, $mysoc, $tva_npr);
 
+	// Customer without VAT
+	if ($customer->tva_assuj==0) $tva_tx=0;
```

Comment:
If customer X has ->tva_assuj, it means this thirdparty X will not include VAT when he will make an invoice (so it is designed for use with vendor invoices).